data/remnant/remnant jobs.txt

```diff
@@ -223,6 +224,115 @@ mission "Remnant: Bounty 5"
 
 
 
+mission "Remnant: Logistics Civil Engineering"
+	name "Civil delivery to <planet>"
+	description `Engineers on <planet> need <cargo> delivered. Logistics adjustment is <payment>.`
+	cargo "remnant civil" 30 80
+	job
+	repeat
+	to offer
+		not "Remnant: Ssil Vida Active"
+		not "flagship planet: Ssil Vida"
+		has "Remnant: Logistics 1: done"
+		random < 51
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+		not distance 0
+		not planet "Ssil Vida"
+		not planet "Esquiline"
+	on visit 
+		dialog phrase "generic cargo on visit"
+	on complete
+		dialog `A project engineer inspects the <cargo> and signs off on your payment of <payment>.`
+		payment 10000 15
+
+
+
+mission "Remnant: Logistics Industrial Delivery"
+	name "Industrial delivery to <planet>"
+	description `Project teams on <planet> need <cargo> delivered. Logistics adjustment is <payment>.`
+	cargo "remnant industrial" 30 300
+	job
+	repeat
+	to offer
+		not "Remnant: Ssil Vida Active"
+		not "flagship planet: Ssil Vida"
+		has "Remnant: Logistics 1: done"
+		has "Remnant: Logistics Civil Engineering: done" >= 9
+		random < 76
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+		not distance 0
+		not planet "Ssil Vida"
+		not planet "Esquiline"
+	on visit 
+		dialog phrase "generic cargo on visit"
+	on complete
+		dialog `A project technician inspects the <cargo> and approves your payment of <payment>.`
+		payment 20000 25
+
+
+
+mission "Remnant: Logistics Outfitter Delivery"
+	name "Outfitter component delivery to <planet>"
+	description `Outfitter repair crews on <planet> need <cargo> parts delivered. Logistics adjustment is <payment>.`
+	cargo "remnant outfitter repair" 30 175
+	job
+	repeat
+	to offer
+		not "Remnant: Ssil Vida Active"
+		not "flagship planet: Ssil Vida"
+		has "Remnant: Logistics 1: done"
+		has "license: Remnant"
+		has "Remnant: Logistics Industrial Delivery: done" >= 9
+		random < 30
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+		not distance 0
+		not planet "Ssil Vida"
+		not planet "Esquiline"
+	on visit 
+		dialog phrase "generic cargo on visit"
+	on complete
+		dialog `A repair technician inspects the <cargo> parts and authorizes your payment of <payment>.`
+		payment 30000 25
+
+
+
+mission "Remnant: Logistics Salvage Delivery"
+	name "Outfitter salvage delivery to <planet>"
```

Comment:
```suggestion
	name "Salvage delivery to <planet>"
```
Same as above. If you can test that yourself and confirm that they fit, feel free to resolve both, but I just don't have time to test it myself right now. Thanks!

Reply:
These mission names do fit, but I think this is a good change, because it makes it clearer that these are distinct jobs.